### PR TITLE
In-Person Payments: extend REST-endpoints documentation

### PR DESCRIPTION
**Pull request by kalessil**
#### Changes proposed in this Pull Request

* Document some knowledge obtained about REST-endpoint used by Mobiles apps.

-------------------

- [x] Run `npm run changelog` to add a changelog file, choose `patch` to leave it empty if the change is not significant. You can add multiple changelog files in one PR by running this command a few times. 
- [x] Covered with tests (or have a good reason not to test in description ☝️)
- [ ] Tested on mobile (or does not apply)

**Post merge**

- [ ] Link to testing instructions from [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) following [these instructions](https://github.com/Automattic/woocommerce-payments/wiki/How-to-write-good-manual-testing-scenarios) : _Add link here / 'QA Testing Not Applicable'_

### CODE REVIEW

File: includes/admin/class-wc-rest-payments-orders-controller.php [hunk 1]
```diff
@@ -192,6 +193,7 @@ public function capture_terminal_payment( $request ) {
 
 	/**
 	 * Returns customer id from order. Create or update customer if needed.
+	 * Use-cases: older versions of Mobile apps got to use this in their workflows.
```
Comment (anu-rock) [inline includes/admin/class-wc-rest-payments-orders-controller.php:196]: Minor nitpick:

```suggestion
	 * Use-cases: This was used by older versions of our Mobile apps in their workflows.
```

File: includes/admin/class-wc-rest-payments-orders-controller.php [hunk 2]
```diff
@@ -245,6 +247,7 @@ public function create_customer( $request ) {
 
 	/**
 	 * Create a new in-person payment intent for the given order ID without confirming it.
+	 * Use-cases: Mobile apps using it for `card_present` payment types. (`interac_present` is handled on appsa side via Stripe SDK).
```
Comment (anu-rock) [inline includes/admin/class-wc-rest-payments-orders-controller.php:250]: Minor nitpick:

```suggestion
	 * Use-cases: Mobile apps using it for `card_present` payment types. (`interac_present` is handled by the apps via Stripe SDK).
```

**Review by anu-rock — APPROVED** [review]
Thanks for updating the doc comments with use cases. This is a nifty initiative.

I've left a couple of comments.

**Comment by kalessil** [comment]
Thanks for the review @anu-rock: I applied the suggested changes and will merge the PR as CI checks pass.

**Pull request merged**